Tasks/SonarQubePreBuild/SonarQubePreBuildImpl.ps1

```diff
@@ -138,9 +138,6 @@ function UpdateArgsForPullRequestAnalysis($cmdLineArgs)
         {
             $cmdLineArgs = $cmdLineArgs + " " + "/d:sonar.analysis.mode=incremental"
         }
-
-		#use this variable in post-test task
-		SetTaskContextVariable "MSBuild.SonarQube.AnalysisModeIsIncremental" "true"
```

Comment:
Useless code, but in any case we can just check if the build is a PR build in the post task and not use this mechanism